src/components/summary-list/template.njk

```diff
@@ -1,5 +1,5 @@
 {%- macro _actionLink(action) %}
-  <a class="govuk-link {%- if action.classes %} {{ action.classes }}{% endif %}" href="{{ action.href }}">
```

Comment:
This is looking really good, do we need to add some documentation in for this new option? In the 'yaml' file?

Reply:
Ah good point! Added